protostar/test_runner/test_environment_exceptions.py

```diff
@@ -1,14 +1,15 @@
 import re
 from abc import ABC, abstractmethod
-from typing import Dict, List, Optional, Union, TypeVar, Type, Any
+from typing import Any, Dict, List, Optional, Type, TypeVar, Union
 
 from starkware.starknet.business_logic.execution.objects import Event
 from typing_extensions import Literal
 
-from protostar.commands.test.expected_event import ExpectedEvent
 from protostar.starknet.hint_local import HintLocal
 from protostar.utils.log_color_provider import SupportedColorName, log_color_provider
 
+from .expected_event import ExpectedEvent
```

Comment:
ReportedException, CheatcodeException and some more should go to `protostar.starknet` module. They are also used in the migrator!

Reply:
Is this resolved?

Reply:
Marking as resolved because PR is approved.